pkg/ui/src/InventoryTable/InventoryTableRow.svelte

```diff
@@ -0,0 +1,34 @@
+<script lang="ts">
+	import type { TableData } from './types';
+	export let isbn;
+	export let title;
+	export let author;
+	export let editor;
+	export let quantity;
+	export let price;
+	export let year;
+	export let outOfPrint;
+	export let editedBy;
+	export let classification;
```

Comment:
Could you, please initialise the values. That makes them optional. You can do `undefined` for each.